src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const DAYS_PROVIDED = days;
+  const DAILY_PRICE = 40;
+  const SMALL_DISCOUNT = 20;
+  const BIG_DISCOUNT = 50;
+
+  if (DAYS_PROVIDED < 3) {
+    result = DAYS_PROVIDED * DAILY_PRICE;
+  } else if (DAYS_PROVIDED >= 3 && DAYS_PROVIDED < 7) {
+    result = (DAYS_PROVIDED * DAILY_PRICE) - SMALL_DISCOUNT;
+  } else if (DAYS_PROVIDED >= 7) {
+    result = (DAYS_PROVIDED * DAILY_PRICE) - BIG_DISCOUNT;
+  }
+
+  return result;
```

Comment:
You need only two if statements. Remember about the checklist (point 4. - Prefer if with return over if else to simplify later conditions).
Simply check if it is a long term or short term. By the way, create constants for 3 & 7 (SHORT_TERM & LONG_TERM)

```suggestion
  if (DAYS_PROVIDED < 3) {
    result = DAYS_PROVIDED * DAILY_PRICE;
  } else if (DAYS_PROVIDED >= 3 && DAYS_PROVIDED < 7) {
   //example
    return = BASE_PRICE - SMALL_DISCOUNT;
  } else if (DAYS_PROVIDED >= 7) {
    result = (DAYS_PROVIDED * DAILY_PRICE) - BIG_DISCOUNT;
  }
  // You are returning this at the end as a default
  return BASE_PRICE;
```